README.md

```diff
@@ -31,3 +31,64 @@ Outputs will be built in the `lib` directory in their own workspaces
 ## Demos
 
 [MVP DApp](./packages/samples/mvp-dapp)
+
+## Quick start
+
+#### Environment
+
+    node >= 18
+
+#### Kuai Installation
+
+First, install `kuai` globally using `npm` or `yarn`, or just use `npx`:
+
+```bash
+# Use npm to install CLI
+$ npm install -g kuai
+
+# OR Use yarn to install CLI
+$ yarn global add kuai
+```
```

Comment:
After a second thought, I found this section is not available now, because 
1. `kuai-cli` is not published yet
2. `kuai-cli` won't be named `kuai` directly, it should have a npm scope, as coded at https://github.com/ckb-js/kuai/blob/develop/packages/cli/package.json#L2

I would suggest adding `npm link` here

Ref: https://docs.npmjs.com/cli/v8/commands/npm-link